lamindb/models/history.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+
+
+class History(models.Model):
+    """Stores the edit history for all LaminDB tables."""
+
+    seqno = models.AutoField(primary_key=True)
```

Comment:
I still don't understand how an auto-incrementing integer helps in presence of two copies of the database that are separately evolving. Let's discuss this on a call tomorrow morning! ☺️

Reply:
Really what we want is some way of ordering history records that's not dependent on, say, the resolution of the record's `created_at` timestamp (since multiple records could be created in the same milliseconds). Making that sequence number auto-increment is an easy way to solve this if you've got separate histories on each node since sequence numbers only need to be local to a given node in that case. If we're sharing history between nodes, we may want to make this non-auto-incrementing and set it to `MAX(seqno) + 1` every time we add a record.

Reply:
Ok!

Reply:
Let's double-check: the sequences in node 1 & 2 increment independently from one another. If we backfill the history table of 1 with new records present in 2, the history table in node 1 will start to deviate from the one in node 2. Isn't this a problem?

--

If it's no problem, we should use a `BigAutoField` because the history table will be the largest table and might exceed 2B rows at some point.

Reply:
After discussion with @alexras I now understand that this is no issue because we're not replaying this field.